foundry/applications/updateNotes.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * The client side Updater application
+ * This displays the progress of patching/update progress for the VTT
+ */
+declare class UpdateNotes extends Application {
+  constructor(target: UpdateNotes['target'], options?: Partial<Application.Options>);
+
+  target: UpdateNotes.Target;
+
+  /**
+   * @override
+   * @defaultValue
+   * ```typescript
+   * mergeObject(super.defaultOptions, {
+   *   id: "update-notes",
+   *   template: "templates/setup/update-notes.html",
+   *   width: 600
+   * });
+   * ```
+   */
+  static get defaultOptions(): Application.Options;
+
+  /** @override */
+  get title(): string;
+
+  /**
+   * @param options - (unused)
+   * @override
+   */
+  getData(options?: Application.RenderOptions): Promise<{ notes: UpdateNotes['target']['notes'] }>;
+
+  /** @override */
+  activateListeners(html: JQuery): void;
+}
+
+declare namespace UpdateNotes {
+  interface Target {
+    notes: any;
+    version: any;
```

Comment:
`version` should be a `string`

Reply:
My original thinking was it can be anything, because anything can be used with string interpolation, but yeah string might be less misleading.